client/spec/search_spec.js

```diff
@@ -39,6 +39,33 @@ describe('Search', function() {
         expect(element.all(by.repeater('items._items')).count()).toBe(1);
         expect(element.all(by.repeater('parameter in tags.selectedKeywords')).count()).toBe(1);
     });
+    
+    it('can search by subject codes field', function() {
+        workspace.switchToDesk('SPORTS DESK').then(content.setListView);
```

Comment:
I had a problem with expected result and because of that I create structure with then() ... Test take result before it completes filtering and still displaying 2 items (it need to be 0, because there are no specified subject codes)